dev-system/dev/user.clj

```diff
@@ -29,6 +32,26 @@
           (fn [app-map]
             (u/map-values #(assoc-in % [:log :level] level) app-map))))
 
+(defn set-mode!
```

Comment:
So if I call `set-mode!` then I can just refresh and the system will restart int that mode, correct?